### ISSUE TITLE
Step 2: change order to match WCAG-EM

### ISSUE THREAD

**Issue by nitedog** [w3c/wcag-em-report-tool#71]
order should be:
- common web pages
- essential functionality
- variety of web pages
- technologies relied upon
- other relevant web pages
